packages/design/sass/_uswds-custom-styles.scss

```diff
@@ -77,15 +77,15 @@ main {
   width: 100%;
 
   .column1 {
-    width: 50%;
+    width: 33.3333%;
   }
 
   .column2 {
-    width: 30%;
+    width: 33.3333%;
   }
 
   .column3 {
-    width: 20%;
+    width: 33.3333%;
```

Comment:
Would it work if this was done with the `grid-col-4` utility?

Reply:
Probably so. I just tweaked what was there and wasn't ejecting out of the custom CSS that's in the design package yet. I'll revisit when I work on making the UI full width.